UnityProject/Packages/packages-lock.json

```diff
@@ -262,11 +262,11 @@
       "url": "https://packages.unity.com"
     },
     "org.unitystation.spacemirror": {
-      "version": "https://github.com/unitystation/Mirror.git?path=/Assets/Mirror#master",
+      "version": "https://github.com/unitystation/Mirror.git?path=/Assets/Mirror#updatingv2",
       "depth": 0,
       "source": "git",
       "dependencies": {},
-      "hash": "82571ae0da21ccfc3fb1fcac7f25890ddc5fbd05"
+      "hash": ""
```

Comment:
remember to generate and commit the generated hash

Reply:
I will do it, it's just a bit fiddly for me since I'm on a newer version of Unity , and it tries updating the packages as well